### DISCUSSION TITLE
[Feature] Improve mobile offline capabilities

### DISCUSSION THREAD

**Discussion by radh21301** [immich-app/immich#4076] [Feature Request]
### The feature

Ignore or feel free to close this request if it is a duplicate. 

I would love it if the mobile app in offline mode can have the same or similar features as in the online one. I use the app in offline mode most of the time (whenever I leave the house). I am thinking of the following features in the search tab:
- maps view
- faces

Is it even possible to do something like this?

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1605

**Reply by radh21301** [reply]
I am not sure how they are related. This request is specifically for offline functionality.

Right now, you cant search for faces or look at the map view of all your assets when offline. I presume this would also need an offline caching of all the assets to enable such functionality.
